homeassistant/components/wemo/wemo_device.py

```diff
@@ -81,7 +81,9 @@ async def async_register_device(
     if device.supports_long_press:
         try:
             await hass.async_add_executor_job(wemo.ensure_long_press_virtual_device)
-        except PyWeMoException:
+        # Temporarily handling all exceptions for #52996 & pywemo/pywemo/issues/276
+        # Replace this with `except: PyWeMoException` after upstream has been fixed.
+        except:  # noqa: E722  pylint: disable=bare-except
```

Comment:
A bare except is never ok.
```suggestion
        except Exception:  # pylint: disable=broad-except
```

Reply:
I'd log the exception too to make it easier to troubleshoot.